### PR TITLE
Added support for Vietnamese localization ('vi')

### PR DESCRIPTION
**Pull request by trinhvuduc**

### CODE REVIEW

**Review by simonpham — APPROVED** [review]
@zulfahmi93 please consider merging this PR 🙌

File: lib/src/l10n/month_year_picker_localizations_vi.dart [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import 'month_year_picker_localizations.dart';
+
+/// The translations for English (`en`).
```
Comment (Phuc-Vuong) [inline lib/src/l10n/month_year_picker_localizations_vi.dart:3]: should it be "for Vietnamese (`vi`)"

Comment (trinhvuduc) [reply lib/src/l10n/month_year_picker_localizations_vi.dart]: Added. Thanks mate

**Comment by zulfahmi93** [comment]
Merged with this [commit](https://github.com/zulfahmi93/month-year-picker/commit/49b325abcd0ca5be061e88ec90c368f1798a1a83).

**Pull request closed without merging**